src/components/ModalPortal.js

```diff
@@ -379,6 +379,7 @@ export default class ModalPortal extends Component {
       onClick: this.handleContentOnClick,
       role: this.props.role,
       "aria-label": this.props.contentLabel,
+      "aria-modal": true,
       ...this.attributesFromObject("aria", this.props.aria || {}),
```

Comment:
```suggestion
      ...this.attributesFromObject("aria",  { modal: true, ...this.props.aria}),
```

This would allow the user to specify in the aria attribute.

Reply:
Ah ok. Yes it's a good idea :)